composer.json

```diff
@@ -25,43 +31,33 @@
         "source": "https://github.com/contao-community-alliance/meta-palettes"
     },
     "require": {
-        "php": ">=5.3",
-        "contao/core-bundle": "~3.0 || ~4.2",
-        "contao-community-alliance/composer-plugin": "~2.4 || ~3.0"
+        "php": ">=5.6",
+        "contao/core-bundle": "~4.4"
     },
     "require-dev": {
-        "bit3/php-coding-standard": "@dev",
-        "phpunit/phpunit": ">=3.7.10,<4-dev",
-        "squizlabs/php_codesniffer": "~1.0",
-        "phpmd/phpmd": "~2.0",
-        "contao-community-alliance/dc-general": "~2.0.0@dev"
-    },
-    "replace": {
-        "contao-legacy/metapalettes": "self.version",
-        "bit3/contao-meta-palettes": "self.version"
+        "contao/manager-plugin": "^2.0.0",
+        "phpcq/all-tasks": "~1.0",
```

Comment:
We might want to bump this to 1.2

Reply:
Done in ad0015f240c20cbf57eeab927d1c82afecf4b83c.